data-model/src/main/java/io/micronaut/data/model/Pageable.java

```diff
@@ -232,4 +241,109 @@ public Sort getSort() {
     static @NonNull Pageable unpaged() {
         return UNPAGED;
     }
+
+    /**
+     * Create a new {@link Pageable} for forward pagination given the cursor after which to query.
+     *
+     * @param cursor The cursor
+     * @param page The page number
+     * @param size The page size
+     * @param sort The sorting
+     * @return The pageable
+     */
+    static @NonNull Pageable afterCursor(@NonNull Cursor cursor, int page, int size, @Nullable Sort sort) {
```

Comment:
If you look at the jakarta [afterCursor](https://github.com/jakartaee/data/blob/a37cac5893c9caf0f23bef43e0860c4d20ef8ece/api/src/main/java/jakarta/data/page/PageRequest.java#L155-L167) method, it says that sorting is required, but it can be supplied by annotation, so here we have to put it as a parameter. Generally, I don't know how useful these methods are, as `nextPage()` and `previousPage()` should cover most use cases.